CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+@samenp
```

Comment:
The correct file location in the repo already exists.
https://github.com/opensearch-project/performance-analyzer/blob/main/.github/CODEOWNERS

And I don't think this file's contents are needed there as it already calls out the owning team.

You can remove this file in your revision.

Reply:
Ok, I got confused because some repos use `.github/CODEOWNERS` and some use `CODEOWNERS`.
- https://github.com/opensearch-project/opensearch-build/blob/main/.github/CODEOWNERS
- https://github.com/opensearch-project/project-tools/blob/main/CODEOWNERS

The [ask](https://github.com/opensearch-project/.github/issues/125) is to replace team aliases with individuals so I will update the existing file.
```
4. Add active maintainers to CODEOWNERS.
5. Remove any broad teams from CODEOWNERS.
```